### ISSUE TITLE
De-standardizing the output in LSTM model if the input was not standardized

### ISSUE THREAD

**Issue by akhil14shukla** [tensorflow/tensorflow#50455]
I was using the LSTM model for time series prediction when I observed that the model standardises the input when given non-standard input. While predicting, the output is also standardized, but there seems no way to de-standardize. 
The current solution is to standardize beforehand (using sklearn or any other) and perform the invert operation on the output.

**System information**
- TensorFlow version : 2.5.0

**The feature and the current behaviour.**
There seems no way to de-standardize the output. One way is to add another attribute that de-standardizes when called or make a flag in the model, which tell whether the input is standardised or not, and if it's not, then standardize and return the de-standardized output.

**Change the current API**
I am asking for an attribute addition, at the least.

**Other Info**
The following links might be of some help:
[https://stackoverflow.com/a/49398454/13428777](https://stackoverflow.com/a/49398454/13428777)
[https://stackoverflow.com/questions/44100837/disable-keras-batch-normalization-standardization](https://stackoverflow.com/questions/44100837/disable-keras-batch-normalization-standardization)

**Comment by jvishnuvardhan** [comment]
@akhil14shukla Sorry for late response.

Data normalization is important for any training and for many models. AFAIK, all the models under `keras.applications` normalizes data because it results in better performance. what is your use-case and can you describe more about it.

Please note that Keras development moved to keras-team/keras repo.](https://github.com/keras-team/keras/issues) repository to focus on only keras. Thanks!

If there is any actionable items, please feel free to open a PR in keras-team/keras  repository.
